Documents/Design/ZTag User Wearable Container Diagram

```diff
@@ -0,0 +1,148 @@
+<mxfile host="app.diagrams.net" modified="2023-03-04T21:44:31.820Z" agent="5.0 (Windows)" etag="emzj11SK3La5-8Eu15bw" version="21.0.2" type="github">
+  <diagram name="Page-1" id="esGuOV4Rx_bUTmLkC89K">
```

Comment:
![ZTag User Wearable Container Diagram drawio](https://user-images.githubusercontent.com/11888681/222931244-ff551ce1-5601-42eb-91fe-d5bd922cc7b8.png)

I approve this Diagram, Looks good.